test/Microsoft.ML.StaticPipelineTesting/Training.cs

```diff
@@ -48,7 +48,7 @@ public void SdcaRegression()
 
             var est = reader.MakeNewEstimator()
                 .Append(r => (r.label, score: ctx.Trainers.Sdca(r.label, r.features, maxIterations: 2,
-                onFit: p => pred = p, advancedSettings: s => s.NumThreads = 1)));
```

Comment:
>NumThreads [](start = 63, length = 10)

Would be good to have numthreads to 1.

Reply:
fixed

---
In reply to: [248104144](https://github.com/dotnet/machinelearning/pull/2093#discussion_r248104144) [](ancestors = 248104144)